modules/rostests/kmtests/ntos_cc/CcCopyRead_drv.c

```diff
@@ -62,10 +65,10 @@ TestEntry(
     KmtRegisterIrpHandler(IRP_MJ_CREATE, NULL, TestIrpHandler);
     KmtRegisterIrpHandler(IRP_MJ_READ, NULL, TestIrpHandler);
 
+    TestFastIoDispatch.SizeOfFastIoDispatch = sizeof(TestFastIoDispatch);
```

Comment:
This is a test. The test shows that the kernel code being tested doesn't check `SizeOfFastIoDispatch`.